test/potato_cluster_test.erl

```diff
@@ -0,0 +1,74 @@
+-module(potato_cluster_test).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("stdlib/include/assert.hrl").
+
+-include("potato_records.hrl").
+
+
+start_stop_test() ->
+    JsonFileName = "test/test_config_3.json",
+
+    {ok, FileData} = file:read_file(JsonFileName),
+
+    JsonConf = jsx:decode(FileData, [return_maps]),
+
+    ForwardFn = fun(_, _, _) -> ok end,
+
+    Data = potato_cluster:start_cluster_from_json(JsonConf, ForwardFn),
+
+    potato_cluster:stop_cluster(Data),
+
```

Comment:
maybe verify that everything got cleaned up properly?

Reply:
no exceptions = good enough

also I don't know how to do that :scream_cat:

Reply:
yeah I don't know how either 😱